src/client/components/App/tests/App.test.js

```diff
@@ -7,9 +7,6 @@ import App from '../index';
 jest.mock('axios');
 jest.mock('../../../services/ltik');
 
-beforeEach(() => {
```

Comment:
@Auroro0804 Are you sure this was causing problems? It really should just be only the yarn.lock changes that broken things